src/algo/thermal-loop/l500-thermal-loop.h

```diff
@@ -0,0 +1,80 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <vector>
+#include "../../types.h"
+
+namespace librealsense {
+namespace algo {
+namespace thermal_loop {
+namespace l500 {
+
+using fx_fy = std::pair< double, double >;
+
+#pragma pack( push, 1 )
+
+class l500_thermal_loop
+{
+public:
+    struct rgb_thermal_calib_info
```

Comment:
If this is a table, why not `thermal_calibration_table`? Why the `rgb`? Is there another?

Reply:
Also, why inside the `l500_thermal_loop` class?

Reply:
Needs documentation, too -- point to the Confluence page for it.
State the expected size (number of temps).

Reply:
Done